### DISCUSSION TITLE
Removing scenery while invisible

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16471] [Feature requests] [answered]
It would be very helpful if you changed the game so when scenery is hidden from view right clicking will not erase it. When building underground rides or pathing, i sometimes delete scenery accidentally when i want to select a track piece. I am not sure of the downside if someone could code this into the game

**Comment by MASHzj** [comment, marked as answer]
This can be done now via View Options > Transparency options. If you set scenery to invisible with the eye-icon, you can't delete it while invisible.
